src/Compilers/CSharp/Test/Emit2/Attributes/InternalsVisibleToAndStrongNameTests.cs

```diff
@@ -178,6 +184,22 @@ public void PubKeyFromKeyFileAttribute_AssemblyKeyFileResolver_RelativeToCurrent
             Assert.True(ByteSequenceComparer.Equals(s_publicKey, comp.Assembly.Identity.PublicKey));
         }
 
+        [Fact]
+        public void PubKeyFromKeyFileAttribute_SigningTempPathNotAvailable()
```

Comment:
Is there a corresponding test I should refer to where the signing temp path *is* available?

Reply:
There are but it's not as clear. Added one here that is much clearer opposite of this one.